src/client/src/consumer.rs

```diff
@@ -579,22 +580,13 @@ impl Record {
         self.offset
     }
 
-    pub fn to_string_lossy(&self) -> String {
-        String::from_utf8_lossy(self.as_ref()).to_string()
+    /// Returns the contents of this Record's key, if it exists
+    pub fn key(&self) -> Option<&[u8]> {
+        self.record.key.as_ref().map(|it| it.as_ref())
     }
-}
 
-impl AsRef<[u8]> for Record {
```

Comment:
This is a breaking change so update the client `Cargo.toml` to `0.6.0` as the node client uses `as_ref`.

Reply:
Yes, you're right. I would prefer to go ahead and make this change and bump the version to 0.7.0 unless anybody objects